File: code_practise/C++/基础/9函数提高/3.cpp
//函数重载
//定义的函数名字可以相同,作用在同一个作用域
//要求函数参数类型、个数、或者顺序不同
//函数的返回值不作为条件
#include<iostream>
using namespace std;
void func(){
    cout<<"hello world"<<endl;
}
void func(int a){
    cout<<"hello world!"<<endl;
}
void func(double a){
    cout<<"Helloworld"<<endl;
}
void func(double a, int b){
    cout<<"Hello world"<<endl;
}
void func(int a, double b){
    cout<<"Hello world!"<<endl;
}
int main(){
    func();
    func(10);
    func(10.0); 
    func(10.0,10);
    func(10,10.0);
    return 0;
}

File: code_practise/python/2/3.py
a=[1,2,3,4,5,6,7,8,9]
print(0 in a)
print(1 in a)               #in运算符判断列表是否有要查找的字符，有输出true否则false

my_list = ["apple", "banana", "cherry"]
fruit = "banana"
if fruit in my_list:
    print(f"{fruit} is in the list.")               #f是格式化字符串字面值，相当于%s那类，一个f对应多个变量，无论{fruit}在哪里都行
else:
    print(f"{fruit} is not in the list.")

my_string = "Hello, World!"
print("W" in my_string)                     #检查字符是否在字符串，也可以检查子字符串是否在

my_set = {1, 2, 3}
print(3 in my_set)                          #也用于集合中